app-android/src/androidTest/java/com/ustadmobile/port/android/screen/ContentEntryEditScreen.kt

```diff
@@ -86,18 +93,11 @@ object ContentEntryEditScreen : KScreen<ContentEntryEditScreen>() {
         repo.clientId
         fragmentScenario.clickOptionMenu(R.id.menu_done)
 
-        val entries = dbRule.db.contentEntryDao.findAllLive().waitUntilWithFragmentScenario(fragmentScenario, 15000) {
+        val entries = dbRule.repo.contentEntryDao.findAllLive().waitUntilWithFragmentScenario(fragmentScenario, 15000) {
```

Comment:
This might not be accurate, and it would get everything out of the database. If something from before had already added any content entry, it would return immediately.

 I would suggest changing this so it looks for the title that was just added. It might also be good to have the title as a parameter of this function.